Gemfile.lock

```diff
@@ -346,4 +351,4 @@ DEPENDENCIES
   xcpretty-travis-formatter
 
 BUNDLED WITH
-   2.2.25
+   2.2.27
```

Comment:
@koke Just noticed that this had been changed... presumably because of doing this on an M1, though I am a bit surprised because I thought I got the right versions of Ruby and Bundler. I'll try to figure out what's going on here.